liquibase-core-new/pom.xml

```diff
@@ -0,0 +1,347 @@
+				<configuration>
+					<artifactSet>
+						<includes>
+                            <include>${project.groupId}:liquibase-core</include>
+                            <include>${project.groupId}:liquibase-cli</include>
+                            <include>${project.groupId}:liquibase-snowflake</include>
+						</includes>
+					</artifactSet>
+					<filters>
+						<filter>
+                            <artifact>*:*</artifact>
+							<excludes>
+                                <exclude>META-INF/maven/**</exclude>
+							</excludes>
+						</filter>
+					</filters>
+                    <createDependencyReducedPom>false</createDependencyReducedPom>
+                    <createSourcesJar>true</createSourcesJar>
+                    <createTestSourcesJar>false</createTestSourcesJar>
+                    <keepDependenciesWithProvidedScope>false</keepDependenciesWithProvidedScope>
+					<transformers>
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                        <transformer implementation="fr.ms.maven.shade.transformers.MergeManifestResourceTransformer"/>
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+							Use a hand-crafted pom for distribution in this jar vs. what gets generated by maven 
+                            <resource>META-INF/maven/org.liquibase/liquibase-core/pom.xml</resource>
+                            <file>${basedir}/src/main/maven/release.pom.xml</file>
+						</transformer>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                            <resource>liquibase.build.properties</resource>
+						</transformer>
+					</transformers>
+				</configuration>
+				<executions>
+					<execution>
+                        <phase>package</phase>
+						<goals>
+                            <goal>shade</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.5.0</version>
+				<configuration>
+                    <tarLongFileMode>posix</tarLongFileMode>
+                    <finalName>liquibase-${project.version}</finalName>
+                    <appendAssemblyId>false</appendAssemblyId>
+                    <attach>false</attach>
+					<descriptors>
+                        <descriptor>${assemblyConfigFile}</descriptor>
+					</descriptors>
+				</configuration>
+				<executions>
+					<execution>
+                        <id>make-assembly</id>
+                        <phase>package</phase>
+						<goals>
+                            <goal>single</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- install the shaded jar as liquibase-core like the final artifact ends up as  -->
+			<plugin>
+                <artifactId>maven-install-plugin</artifactId>
```

Comment:
remove this and might need to use different logic to install the liquibase-core-new module